enforcer.go

```diff
@@ -334,6 +337,11 @@ func (e *Enforcer) EnableAutoSave(autoSave bool) {
 	e.autoSave = autoSave
 }
 
+// EnableAutoSave controls whether to save a policy rule automatically to the adapter when it is added or removed.
```

Comment:
The comment here is incorrect

Reply:
Whoops, my bad. forgot to change it when copying it.
I wrote the comment to match the function name, I will change the description if it is changed.